src/MediaStreamTrack.ts

```diff
@@ -36,6 +39,8 @@ class MediaStreamTrack extends defineCustomEventTarget(...MEDIA_STREAM_TRACK_EVE
         this.id = info.id;
         this.kind = info.kind;
         this.remote = info.remote;
+
+        this._registerEvents();
```

Comment:
We should only do this for local tracks. It's not necessary to release remote tracks, so those will leak.

Reply:
Added in a check on `this.remote` and tested, seems to work fine there. 

A little worried about the SerializeUtils functions always returning true for remote though:

https://github.com/react-native-webrtc/react-native-webrtc/blob/master/android/src/main/java/com/oney/WebRTCModule/SerializeUtils.java#L76

https://github.com/react-native-webrtc/react-native-webrtc/blob/0651e2b0600435dacd7b1e13d5f47914252e1e35/ios/RCTWebRTC/SerializeUtils.m#L174

I don't think there's a way for this flag to be incorrectly set for local tracks atm though, so should be fine if this isn't misused.

Reply:
Hum good point, we should check that.